composer.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "surfnet/stepup-gssp-example",
+  "name": "surfnet/stepup-webauthn",
   "license": "Apache-2.0",
-  "description": "Example Generic SAML Stepup Provider.",
+  "description": "Webauthn SAML Stepup Provider.",
```

Comment:
You stripped `Generic` even tho this is the G part of GSSP. You might want to put it back :)

Reply:
Also I think you should write WebAuthn instead of Webauthn

Reply:
🥇